console/src/main/java/com/alibaba/nacos/console/security/nacos/JwtTokenManager.java

```diff
@@ -97,6 +100,13 @@ public Authentication getAuthentication(String token) {
      * @param token token
      */
     public void validateToken(String token) {
-        Jwts.parser().setSigningKey(authConfigs.getSecretKey()).parseClaimsJws(token);
+        makeSureSecretKeyBytes();
+        Jwts.parserBuilder().setSigningKey(authConfigs.getSecretKeyBytes()).build().parseClaimsJws(token);
+    }
+    
+    private void makeSureSecretKeyBytes() {
+        if (authConfigs.getSecretKeyBytes() == null) {
+            authConfigs.setSecretKeyBytes(Decoders.BASE64.decode(authConfigs.getSecretKey()));
```

Comment:
Can We remove this Base64？

Reply:
> Can We remove this Base64？

sure, we just ensure the secretBytes is same in the process that do parseClaimsJws and compact